symfony/client/src/orangehrmMaintenancePlugin/components/EmployeeRecords.vue

```diff
@@ -66,15 +62,18 @@ export default {
   props: {
     includeEmployeesParam: {
       type: String,
-      required: true,
+      required: false,
+      default: 'currentAndPast',
     },
     titleLabel: {
       type: String,
       required: true,
+      default: 'Employee Name',
     },
     autocompleteLabel: {
       type: String,
-      required: true,
+      required: false,
+      default: '',
```

Comment:
in my opinion better not to set a default here since right now this component is only used in two places and since they both pass in different values for all 3 props there is no "default" value. Maybe once purge candidates is introduced we could set default. Right now my idea is to keep as it was before. Remove the `default` and have `required: true`

Reply:
i just only put for reusable scenarios, if we don't needed then i remove it